lib/dai-plugin-mcd/src/CdpType.js

```diff
@@ -13,6 +13,14 @@ export default class CdpType {
     this._ilkBytes = getIlkForCurrency(currency);
   }
 
+  get ilkId() {
```

Comment:
Since we're not doing anything special in this getter, you can just do `this.ilkId = getIlkForCurrency(currency)` in the constructor and skip defining the getter entirely